Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -241,9 +241,9 @@ std::vector<RedumpVerifier::PotentialMatch> RedumpVerifier::ScanDatfile(const st
       continue;
 
     const std::string serials = game.child("serial").text().as_string();
-    if (serials.empty())
+    if (serials.empty() || StringBeginsWith(serials, "DS"))
```

Comment:
No access to `std::string::startswith` yet I assume?

Reply:
No, we're on C++17 currently.